lib/onebox/engine/google_docs_onebox.rb

```diff
@@ -45,20 +50,6 @@ def shorttype
       def match
         @match ||= @url.match(@@matcher)
       end
-
-      def get_og_data
-        response = Onebox::Helpers.fetch_response(url, 10) rescue nil
```

Comment:
The response - Onebox::Helpers.fetch_response(url,  10) rescue nil 
should that be in the code?  I am java and I do not see nil ever?